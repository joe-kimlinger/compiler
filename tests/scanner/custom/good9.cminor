// length
// length = 253 + 2 quotes and null = 256
s1: string = "1234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123"
// Same string as above, with every character escaped
s2: string = "\1\2\3\4\5\6\7\8\9\o\1\2\3\4\5\6\7\8\9\o\1\2\3\4\5\6\7\8\9\o\1\2\3\4\5\6\7\8\9\o\1\2\3\4\5\6\7\8\9\o\1\2\3\4\5\6\7\8\9\o\1\2\3\4\5\6\7\8\9\o\1\2\3\4\5\6\7\8\9\o\1\2\3\4\5\6\7\8\9\o\1\2\3\4\5\6\7\8\9\o\1\2\3\4\5\6\7\8\9\o\1\2\3\4\5\6\7\8\9\o\1\2\3\4\5\6\7\8\9\o\1\2\3\4\5\6\7\8\9\o\1\2\3\4\5\6\7\8\9\o\1\2\3\4\5\6\7\8\9\o\1\2\3\4\5\6\7\8\9\o\1\2\3\4\5\6\7\8\9\o\1\2\3\4\5\6\7\8\9\o\1\2\3\4\5\6\7\8\9\o\1\2\3\4\5\6\7\8\9\o\1\2\3\4\5\6\7\8\9\o\1\2\3\4\5\6\7\8\9\o\1\2\3\4\5\6\7\8\9\o\1\2\3\4\5\6\7\8\9\o\1\2\3"
