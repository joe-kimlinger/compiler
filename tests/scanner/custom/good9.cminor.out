COMMENT
COMMENT
IDENTIFIER s1
COLON
STRING
ASSIGN
STRING_LITERAL 1234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123
COMMENT
IDENTIFIER s2
COLON
STRING
ASSIGN
STRING_LITERAL 123456789o123456789o123456789o123456789o123456789o123456789o123456789o123456789o123456789o123456789o123456789o123456789o123456789o123456789o123456789o123456789o123456789o123456789o123456789o123456789o123456789o123456789o123456789o123456789o123456789o123
